packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -29,7 +29,7 @@ bool _shouldRunPubGet({ File pubSpecYaml, File dotPackages }) {
   return false;
 }
 
-Future<Null> pubGet({
+Future<Null> pubGet(String pubContext, {
```

Comment:
to make the call sites clearer about whether the string represents a command to `pub` or just metadata, can you make this an `@required` named argument?

Reply:
+1, and maybe just call it `context`?

Reply:
ack